invesalius/net/neuronavigation_api.py

```diff
@@ -48,3 +53,28 @@ def update_coil_pose(self, position, orientation):
                 position=position,
                 orientation=orientation,
             )
+
+    def update_coil_mesh(self, polydata):
+        wrapped = dataset_adapter.WrapDataObject(polydata)
+
+        points = np.array(wrapped.Points)
+        polygons_raw = np.array(wrapped.Polygons)
+
+        # The polygons are returned as 1d-array of the form
+        #
+        # [n_0, id_0(0), id_0(1), ..., id_0(n_0),
+        #  n_1, id_1(0), id_1(1), ..., id_1(n_1),
+        #  ...]
+        #
+        # where n_i is the number of vertices in polygon i, and id_i's are indices to the vertex list.
+        #
+        # Assert that all polygons have an equal number of vertices, reshape the array, and drop n_i's.
+        #
+        assert all(polygons_raw[0::self.N_VERTICES_IN_POLYGON + 1] == self.N_VERTICES_IN_POLYGON)
```

Comment:
I think it's better and faster to `np.all` than `all`, see:

```python
>>> q  = np.ones(100)
>>> %timeit all(q)
2.26 µs ± 30 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)
>>> %timeit np.all(q)
1.34 µs ± 9.82 ns per loop (mean ± std. dev. of 7 runs, 1000000 loops each)
```

Reply:
Done. Thanks for the comments, I learned something new about numpy while at it.